templates/zerver/integrations/openshift.md

```diff
@@ -1,43 +1,43 @@
-repository, and push it to your app.
+1.  Then, create a new commit including all the changes made to the
+    repository, and push it to your app.
 
-After that, connect to the application through SSH. If you don’t know
-how to do this, log in to your OpenShift Online account, go to your
-application’s dashboard, and click **Want to log in to your
-application?**.  There you’ll find the app’s SSH user, address, and
-further information on SSH, in case you need it.
+1.  After that, connect to the application through SSH. If you don’t know
+    how to do this, log in to your OpenShift Online account, go to your
+    application’s dashboard, and click **Want to log in to your
+    application?**.  There you’ll find the app’s SSH user, address, and
+    further information on SSH, in case you need it.
 
-![Connecting to application](/static/images/integrations/openshift/002.png)
+    ![Connecting to application](/static/images/integrations/openshift/002.png)
 
-{!change-zulip-config-file.md!}
+1.  {!change-zulip-config-file-indented.md!}
 
-You can also specify which pushes will result in notifications and to
-what stream the notifications will be sent by modifying the
-`deployment_notice_destination` function in
-`zulip_openshift_config.py`. By default, deployments triggered by
-commits pushed to the `master` and `test-post-receive` branches will
-result in a notification to stream `deployments`.
+1.  You can also specify which pushes will result in notifications and to
+    what stream the notifications will be sent by modifying the
+    `deployment_notice_destination` function in
+    `zulip_openshift_config.py`. By default, deployments triggered by
+    commits pushed to the `master` and `test-post-receive` branches will
+    result in a notification to stream `deployments`.
 
-Save the file, and symlink
-`$OPENSHIFT_PYTHON_DIR/virtenv/share/zulip/integrations/openshift/post-receive`
-into the `~/app-root/repo/.openshift/action_hooks` directory.
+1.  Save the file, and symlink
+    `$OPENSHIFT_PYTHON_DIR/virtenv/share/zulip/integrations/openshift/post-receive`
+    into the `~/app-root/repo/.openshift/action_hooks` directory.
 
-Whenever you make a push to the `master` branch of your application’s
-repository (or whichever branch you configured above), or if you force
-a deployment, the Zulip OpenShift plugin will send an automated
-notification.
+1.  Whenever you make a push to the `master` branch of your application’s
+    repository (or whichever branch you configured above), or if you force
+    a deployment, the Zulip OpenShift plugin will send an automated
+    notification.
 
 {!congrats.md!}
 
 ![OpenShift integration message](/static/images/integrations/openshift/001.png)
 
-**Testing**
+### Testing
 
-You can test the plugin without changing your `master` branch by pushing to the `test-post-receive` branch.
+1.  You can test the plugin without changing your `master` branch by pushing to the `test-post-receive` branch.
```

Comment:
Same comment on OpenShift for the testing section as with Nagios -- the single-item numbered list feels worse than just having a paragraph.